src/components/Navbar.vue

```diff
@@ -5,6 +5,8 @@
     </router-link>
     <router-link to="/">Home</router-link> 
     <router-link to="/pedidos">Pedidos</router-link>
+    <router-link to="/Cadastro">Cadastro</router-link>
```

Comment:
The routes are defined on lower case, in our projects

Reply:
Ajusted.